src/System.ComponentModel.TypeConverter/src/System/ComponentModel/DecimalConverter.cs

```diff
@@ -31,7 +32,12 @@ public class DecimalConverter : BaseNumberConverter
         /// </summary>
         public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
         {
-            return destinationType == typeof(InstanceDescriptor) || base.CanConvertTo(context, destinationType);
+            if (destinationType == typeof(InstanceDescriptor))
+            {
+                return true;
+            }
+
+            return base.CanConvertTo(context, destinationType);
```

Comment:
Why did you change this?

I don't understand what these stylistic changes have to do with adding more tests as the PR is titled.

Reply:
reverted